code/iaas/config-item/common/src/main/java/io/cattle/platform/configitem/version/impl/ConfigItemStatusManagerImpl.java

```diff
@@ -67,7 +59,7 @@
     AgentLocator agentLocator;
```

Comment:
For the record, I did not explicit load your code in eclipse to be the warning police. I wanted to click through the code and this jumped out at me. :-)